si/orange-canvas.yaml

```diff
@@ -0,0 +1,3536 @@
+        def `is_ows`:
+            .ows: false
+        .: false
+    def `workflows`:
+        tutorials_entry_points: false
+        Could not load examples from %r (%r): false
+        Could not load examples from %r: false
+        'A callable entry point (%r) raised an ': false
+        unexpected error.: false
+    class `ExampleWorkflow`:
+        def `abspath`:
+            cannot resolve resource to an absolute name: false
+        def `stream`:
+            rb: false
+application/outputview.py:
+    class `TerminalView`:
+        def `sizeHint`:
+            X: false
+    class `TerminalTextDocument`:
+        def `__init__`:
+            defaultFont: false
+        TextStream: false
+        Formatter: false
+        def `connectedStreams`:
+            TextStream: false
+        def `connectStream`:
+            TextStream: false
+            '''charformat'' and kwargs cannot be used together': false
+        def `disconnectStream`:
+            TextStream: false
+        def `clone`:
+            TerminalTextDocument: false
+    class `OutputView`:
+        def `formated`:
+            '''Use ''formatted''': false
+    class `Formatter`:
+        def `formated`:
+            Use 'formatted': false
+    class `formater`:
+        def `__init__`:
+            'Deprecated: Renamed to Formatter.': false
+    class `TextStream`:
+        def `write`:
+            write operation on a closed stream.: false
+        def `writelines`:
+            write operation on a closed stream.: false
+        def `flush`:
+            write operation on a closed stream.: false
+        def `detach`:
+            detach: false
+        def `read`:
+            read: false
+        def `readline`:
+            readline: false
+        def `readlines`:
+            readlines: false
+        def `fileno`:
+            fileno: false
+        def `seek`:
+            seek: false
+        def `tell`:
+            tell: false
+    class `ExceptHook`:
+        def `__call__`:
+            ' Exception': false
+            ' (in non-GUI thread)': false
+            '{:-^79}\n': false
+            ' ': false
+            '-': false
+            \n: false
+application/schemeinfo.py:
+    class `SchemeInfoEdit`:
+        def `__setupUi`:
+            untitled: nepoimenovan
+            Title: Naslov
+            Description: Opis
+        def `setScheme`:
+            untitled: nepoimenovan
+        def `commit`:
+            untitled: nepoimenovan
+    class `SchemeInfoDialog`:
+        def `__setupUi`:
+            Workflow Info: Podatki o delotoku
+            <h3>{0}</h3>: false
+            heading: false
+            auto-show-container: false
+            Show when I make a New Workflow.: Pokaži ob sestavljanju novega delotoka
+            auto-show-check: false
+            'You can also edit Workflow Info later ': ' Podatke o delotoku lahko urejate tudi kasneje '
+            (File -> Workflow Info).: (Datoteka -> Podatki o delotoku).
```

Comment:
Tule se u in oklepaj skrijeta "za rob" sporočilnega okna. Verjetno se tudi to razreši z Aleševo magijo?